# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/CanvasRenderer.h
# Commit: 6f19f43e0a0b
# Full Hash: 6f19f43e0a0bad72fa370191f4cc7f86f58566c2
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 09:34:54
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - NON_PREMULT TextureClient iff NON_PREMULT CanvasClient. r=handyman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78800
# ==============================================================================

diff -r 073302d26c5e -r 6f19f43e0a0b gfx/layers/CanvasRenderer.h
--- a/gfx/layers/CanvasRenderer.h	Thu Jun 11 06:37:38 2020 +0000
+++ b/gfx/layers/CanvasRenderer.h	Thu Jun 11 06:37:40 2020 +0000
@@ -137,7 +137,8 @@
     return nullptr;
   }
 
-  std::shared_ptr<BorrowedSourceSurface> BorrowSnapshot() const;
+  std::shared_ptr<BorrowedSourceSurface> BorrowSnapshot(
+      bool requireAlphaPremult = true) const;
 
  protected:
   void FirePreTransactionCallback() const;